src/material/form-field/testing/form-field-harness.ts

```diff
@@ -165,19 +165,35 @@ export class MatFormFieldHarness extends ComponentHarness {
   /**
    * Gets a reference to the container element which contains all projected
    * prefixes of the form-field.
+   * @deprecated Use `getPrefixText` instead.
+   * @breaking-change 11.0.0
    */
   async getHarnessLoaderForPrefix(): Promise<TestElement|null> {
     return this._prefixContainer();
   }
 
+  /** Gets the text inside the prefix element. */
+  async getPrefixText(): Promise<string> {
+    const prefix = await this._prefixContainer();
+    return prefix ? prefix.text() : '';
+  }
+
   /**
    * Gets a reference to the container element which contains all projected
    * suffixes of the form-field.
+   * @deprecated Use `getSuffixText` instead.
```

Comment:
Should we actually just fix this? Seems like this happened by mistake, but the methods actually should exist?

Reply:
The reason I think we didn't do `<..>Text` is that a prefix/suffix doesn't necessarily need to be about text. i.e. it can contain icons or arbitrary content, which can be tested against with the harness loader too?

Reply:
Oh, looks like this has been changed in 1c1af58ced755a248234ed47802ac528f41d46ff as part of Miles' suggestion (https://github.com/angular/components/pull/17874#issuecomment-561858220) but the logic has not been updated properly. Any reason to deprecate the actual method instead of making it return a harness loader?

Reply:
I decided to add the text method since I don't see what else you would be doing with the `TestElement`. We could make it return a harness loader, but at this point it'll be a breaking change.

Reply:
Yeah for sure, it shouldn't be a `TestElement`. It just feels wrong to me to deprecate the method while we actually would want to keep the method (just with the fix IMO). cc. @mmalerba for thoughts on this.

The additional methods you added are a nice addition we could keep anyway I guess.

Reply:
@crisbeto Chatted with Miles. Looks like we can remove this as we'll add the more generic harness loader methods w/ an enum for accessing the prefix/suffix containers. That makes sense to me.